build.gradle.kts

```
@@ -601,8 +608,9 @@ tasks.startShadowScripts {
 /* ******************** HiveMQ composite build ******************** */
 
 tasks.register<Sync>("updateSpecs") {
-    from(hivemqOpenApi) { rename { "hivemq-openapi.yaml" } }
-    from(swarmOpenApi) { rename { "hivemq-swarm-openapi.yaml" } }
+    group = "cli"
```

Every tasks here is part of the cli.
```suggestion
    group = "openapi"
```